rjfoley/style.css

```diff
@@ -0,0 +1,23 @@
+/* Import fonts */
+
+/* Re-styling Basics i.e.: body h1 h2 h3 ... ; */
+
+/* Snippet Title */
+
+.only-text {
+	font-size:24px;
```

Comment:
This is just being nit picky, but try to be consistent about leaving spaces after semicolons. 😄

Reply:
Gotcha. 😉